### ISSUE TITLE
Question about qrnn's output

### ISSUE THREAD

**Issue by hellomotor** [JonathanRaiman/tensorflow_qrnn#1]
it has two output with same shape, which one should be used for next initial_state?

**Comment by AzizCode92** [comment]
both

**Comment by hellomotor** [comment]
@AzizCode92 ,  can you please provide some sample code about how to use tensorflow_qrnn correctly?  i use it for language modeling,  like following: 

          initial_state = tf.zeros([batch_size, hidden_size], dtype=tf.float32)
          outputs, states = qrnn.qrnn(
                        inputs, num_outputs=hidden_size,
                        initial_state=initial_state,
                        time_major=False)
           output = tf.reshape(outputs, [-1, hidden_size])
           final_state = states[:, -1, :]

     then i use final_state feed as next initial_state  and calculate cross entropy loss on output, but i found loss value goes incredible low(1e-3) after some epoch, it must be something wrong.

**Comment by AzizCode92** [comment]
It seems you're correct. 
Try to use the output as following:
```python
prediction = tf.nn.softmax(output)
# Define loss and optimizer (for example)
loss_op = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(
    logits=output, labels=Y))
optimizer = tf.train.GradientDescentOptimizer(learning_rate=learning_rate)
train_op = optimizer.minimize(loss_op)
```
